cmd/cvc-operator/controller/cvc_controller.go

```diff
@@ -274,11 +282,19 @@ func (c *CVCController) createVolumeOperation(cvc *apis.CStorVolumeClaim) (*apis
 		return nil, err
 	}
 
+	// Fetch the volume replica pool names and use them in PDB and updating in
```

Comment:
add a comment in distributePendingCVRs that - it returns error when required number of CVRs are NOT created

Reply:
Done